README.md

```diff
@@ -138,6 +73,7 @@ Please see the [contributing guidelines](CONTRIBUTING.md).
 * [Microsoft Graph](https://graph.microsoft.io)
 * [Office Dev Center](http://dev.office.com/)
 * [Microsoft Graph JavaScript Client Library](https://github.com/microsoftgraph/msgraph-sdk-javascript)
+* [Use the Microsoft Graph SDKs with the beta API](https://docs.microsoft.com/en-us/graph/sdks/use-beta?tabs=CS)
 
 ## Copyright
 Copyright (c) 2017 Microsoft. All rights reserved.
```

Comment:
Might as well update the copyright date. It looks better when updated to this year.

Reply:
Done.